### ISSUE TITLE
make-homepage.rst

### ISSUE THREAD

**Issue by calexo** [symfony-cmf/symfony-cmf-docs#762]
http://symfony.com/doc/current/cmf/tutorial/make-homepage.html

The @Route annotation doesn't work if this piece of code is not in @Bundle/Config/routing.yml
```
default:
    resource: "@DewittCmsBundle/Controller/DefaultController.php"
    type:     annotation
    prefix:   /
```

**Comment by dbu** [comment]
thanks for spotting. see #763 for the fix. i decided to go with routing configuration, as the annotation itself does not work without additional configuration anyways.

**Comment by dantleech** [comment]
@dbu do you know why this has changed? I had run through this tutorial
several times and never encountered this issue.

**Comment by calexo** [comment]
Hi !

Perfect, thanks !

I'll follow #763

**Comment by dbu** [comment]
fixed in #763
